specs/playwright/auto-generate-tests-with-codegen.yaml

```diff
@@ -0,0 +1,11 @@
+name: Auto-generate tests with Codegen
```

Comment:
Small nit: The name of the workflow is what we use to search for workflows within Warp. Given that, I think it will help with discoverability if we include "playwright" in the title of each of these workflows so that users can quickly search for playwright workflows.

For example: "Auto-generate Playwright tests with Codegen"

Reply:
You are right. I'm on it @alokedesai